plugin.go

```diff
@@ -200,6 +202,20 @@ func commandBuild(build Build) *exec.Cmd {
 		args = append(args, "--build-arg", arg)
 	}
 
+	labelSchema := []string{
+		fmt.Sprintf("build-date=%s", time.Now().Format(time.RFC3339)),
+		fmt.Sprintf("vcs-ref=%s", build.Name),
+		fmt.Sprintf("vcs-url=%s", build.Remote),
+	}
+
+	if len(build.LabelSchema) > 0 {
+		labelSchema = append(labelSchema, build.LabelSchema...)
+	}
+
+	for _, label := range labelSchema {
+		args = append(args, "--label", fmt.Sprintf("org.label-schema.%s", label))
```

Comment:
I'm open for ideas, we can also name the flag only labels and allow general label addition, not just labels prefixed with `org.label-schema.`. We can set the 4 labels from above automatically as we know the values for them, otheres have to be set like this in that case:

```
pipeline:
  docker:
    image: plugins/docker
    pull: true
    secrets: [ docker_username, docker_password ]
    repo: foo/foo
    labels:
      - org.label-schema.name=Foo
      - org.label-schema.version=${DRONE_BRANCH}
```